app/uk/gov/hmrc/agentservicesaccount/actions/AuthActions.scala

```diff
@@ -92,36 +90,6 @@ class AuthActions @Inject()(appConfig: AppConfig,
       Left(Forbidden)
   }
 
-  def withFullUserDetails(body: AgentInfo => Future[Result])
-                         (implicit ec: ExecutionContext, hc: HeaderCarrier, request: Request[AnyContent])
-  : Future[Result] = {
-    authorised(AuthProviders(GovernmentGateway))
```

Comment:
I take it that these aren't used anywhere?

Reply:
that's right, I think they got re-added when Alex merged main because of the configuration refactor he did in AuthActions. authActionRefiner & handleFailureRefiner cover their functionality and are part of the ActionBuilders